### PR TITLE
Update tensorflow_backend.py

### PR DESCRIPTION
**Pull request by xiaodi-faith**
using multiple input

### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
Thanks for the PR. Can you give a bit more background about what you are trying to do and what problem you are solving?

**Comment by xiaodi-faith** [comment]
> Thanks for the PR. Can you give a bit more background about what you are trying to do and what problem you are solving?

When the network need more than one input, 
this code 'feed_dict[tensor] = value' return some error, the tensor variable is a list, and it include more than one tensor, like that: tensor = [<tf.Tensor 'input_1:0' ........>, <tf.Tensor 'input_2:0'........>]

**Comment by gabrieldemarmiesse** [comment]
Can you add a unit test that shows the use case which you are fixing?

**Comment by gabrieldemarmiesse** [comment]
I don't think I'm competent enough to review this. From the git blame, it seems that @fchollet is the one who is the most familiar with the `Function` class. When you have the time, could you review this? Thanks.

**Comment by xiaodi-faith** [comment]
> I don't think I'm competent enough to review this. From the git blame, it seems that @fchollet is the one who is the most familiar with the `Function` class. When you have the time, could you review this? Thanks.

I add a new file and pull a  new request '#11255'. I don't know if it's appropriate.

File: keras/backend/tensorflow_backend.py [hunk 1]
```diff
@@ -2686,7 +2686,11 @@ def _legacy_call(self, inputs):
                     (np.expand_dims(sparse_coo.row, 1),
                      np.expand_dims(sparse_coo.col, 1)), 1)
                 value = (indices, sparse_coo.data, sparse_coo.shape)
-            feed_dict[tensor] = value
+            if type(tensor) is list:
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: Use `isinstance` instead of `type`

Comment (xiaodi-faith) [reply keras/backend/tensorflow_backend.py]: I add two commits, one is to create a file to test multi inputs, the other is to change type() to isinstance()

Comment (xiaodi-faith) [reply keras/backend/tensorflow_backend.py]: > Use `isinstance` instead of `type`

I add two commits, one is to create a file to test multi inputs, the other is to change type() to isinstance()

**Comment by fchollet** [comment]
I see your use case now: indeed, this was about using nested list of inputs in `K.function`. This is not something we should support.

In your script, you should replace

`train = K.function([self.model.input, a, y], [loss], updates=updates)`

With

`train = K.function(self.model.input + [a, y], [loss], updates=updates)`
